server/app/views/admin/migration/AdminExportViewPartial.java

```diff
@@ -0,0 +1,66 @@
+package views.admin.migration;
+
+import static j2html.TagCreator.button;
+import static j2html.TagCreator.div;
+import static j2html.TagCreator.form;
+
+import com.google.inject.Inject;
+import controllers.admin.routes;
+import j2html.tags.DomContent;
+import j2html.tags.specialized.DivTag;
+import play.mvc.Http;
+import views.BaseHtmlView;
+import views.components.ButtonStyles;
+import views.components.FieldWithLabel;
+
+/** An HTMX partial for portions of the page rendered by {@link AdminExportView}. */
+public final class AdminExportViewPartial extends BaseHtmlView {
+  @Inject
+  AdminExportViewPartial() {}
+
+  /**
+   * The ID for the div containing the program json preview. Must be applied to the top-level DOM
+   * element of each partial so that replacement works correctly.
+   */
+  public static final String PROGRAM_JSON_ID = "json-preview";
+
+  /** Renders the json preview section. */
+  public DomContent renderProgramData(Http.Request request, String json, String adminName) {
```

Comment:
suggestion (non-blocking):  Could we call this "renderJSONPreview" to be more clear?

Reply:
Oops, yes this was a copy/paste error.